src/components/GameView/GameOverDialog.vue

```diff
@@ -90,6 +124,27 @@ export default {
 
       return 'loss-img';
     },
+    currentMatch(){
+      return this.game.currentMatch
+    },
+    matchGameStats(){
+      if(!this.game.currentMatch?.games) return []
+      const result= this.game.currentMatch.games.map(game=>{
+        if (game.result === -1) return 'I'
+        if(game.result === 0 || game.result === 1){
+          if(game.p0 === this.player.id) return 'W'
+          return 'L'
+        }
+        return 'D'
+      })
+      return result
+    },
```

Comment:
Stay consistent with your coding style, spaces after the if statements, spaced after the equals for const declarations.

Reply:
For all linting related comments, I'm surprised the linting tools didn't catch all these. I'll need to rely on those since I won't be able to catch all these. 

I'll look into my setup on why these are not working even when I ran `npm run lint:fix`. However we should think about getting these to work on any machine out of the box.

Reply:
And this will save us time once we agree to a set of linting rules to respect, and have the code linted consistently

Reply:
Linting is something I plan on improving as part of my arch adjustments. Pinia, linting, typescript. Some sort of rough order there, haha.